v19.2/explain.md

```diff
@@ -252,88 +285,107 @@ To include all details used by the optimizer, including statistics, use `OPT, EN
-      },
-      {
-          "columns": [
-              "stardate"
-          ],
-          "created_at": "2019-03-26 19:49:53.18699+00:00",
-          "distinct_count": 75,
-          "histo_col_type": "",
-          "name": "__auto__",
-          "null_count": 4,
-          "row_count": 79
-      }
-  ]';
-
-  EXPLAIN (OPT, ENV) SELECT * FROM episodes WHERE season > 3 ORDER BY season ASC;
-  ----
-  sort
-   └── select
-        ├── scan episodes
-        └── filters
-             └── season > 3
-(77 rows)
+Version: CockroachDB CCL v19.2.0
+
+CREATE TABLE episodes (
+	id INT8 NOT NULL,
+	season INT8 NULL,
+	num INT8 NULL,
+	title STRING NULL,
+	stardate DECIMAL NULL,
+	CONSTRAINT "primary" PRIMARY KEY (id ASC),
+	FAMILY "primary" (id, season, num, title, stardate)
+);
+
+ALTER TABLE startrek.public.episodes INJECT STATISTICS '[
+    {
+        "columns": [
+            "id"
+        ],
+        "created_at": "2019-10-08 18:29:26.222957+00:00",
+        "distinct_count": 79,
+        "histo_col_type": "INT8",
+        "name": "__auto__",
+        "null_count": 0,
+        "row_count": 79
+    },
+    {
+        "columns": [
+            "season"
+        ],
+        "created_at": "2019-10-08 18:29:26.222957+00:00",
+        "distinct_count": 3,
+        "histo_col_type": "",
+        "name": "__auto__",
+        "null_count": 0,
+        "row_count": 79
+    },
+    {
+        "columns": [
+            "num"
+        ],
+        "created_at": "2019-10-08 18:29:26.222957+00:00",
+        "distinct_count": 29,
+        "histo_col_type": "",
+        "name": "__auto__",
+        "null_count": 0,
+        "row_count": 79
+    },
+    {
+        "columns": [
+            "title"
+        ],
+        "created_at": "2019-10-08 18:29:26.222957+00:00",
+        "distinct_count": 79,
+        "histo_col_type": "",
+        "name": "__auto__",
+        "null_count": 0,
+        "row_count": 79
+    },
+    {
+        "columns": [
+            "stardate"
+        ],
+        "created_at": "2019-10-08 18:29:26.222957+00:00",
+        "distinct_count": 76,
+        "histo_col_type": "",
+        "name": "__auto__",
+        "null_count": 4,
+        "row_count": 79
+    }
+]';
+
+EXPLAIN (OPT, ENV) SELECT * FROM episodes WHERE season > 3 ORDER BY season ASC;
+----
+sort
+ └── select
+      ├── scan episodes
+      └── filters
+           └── season > 3
+~~~
+
+### `VEC` option
+
+The `VEC` option shows details about the [vectorized execution plan](vectorized-execution.html#how-vectorized-execution-works) for the query.
+
+{% include copy-clipboard.html %}
+~~~ sql
+> EXPLAIN (VEC) SELECT * FROM episodes WHERE season > 3 ORDER BY season ASC;
 ~~~
 
+~~~
+                  text
++---------------------------------------+
+  │
+  └ Node 1
+    └ *colexec.sortOp
+      └ *colexec.selGTInt64Int64ConstOp
+        └ *colexec.colBatchScan
+(5 rows)
+~~~
+
+The output shows the different functions that will be used to process each batch of column-oriented data.
+
```

Comment:
AFAICT these are the names of actual Go functions right?  If so I think we should call these "internal functions"